src/modules/secrets/Elsa.Secrets.Http/Services/OAuth2TokenService.cs

```diff
@@ -48,23 +48,25 @@ public async Task<TokenResponse> GetToken(Secret secret, string? authCode, strin
 			{ "grant_type", secret.GetProperty("GrantType") },
 			{ "client_id", clientId },
 			{ "client_secret", clientSecret },
-			{ "scope", secret.GetProperty("Scope") }
+			{ "offline_access ", "true" }
```

Comment:
Hey 👋🏻 

FYI, `offline_access` is not a standard parameter (and not used by OpenIddict unless you add custom logic to handle it). It also seems there's an extra space at the end of the parameter name so it's likely something you'll want to fix/remove 😃

Reply:
Damn, you are right. "offline_access" is probably not required indeed, but was something I added when trying to get refresh tokens to work. I will create a new PR to remove this.

Reply:
@tanelkuhi If you create a PR with the required corrections we may be able to include it in a 2.11.2 release soon